### ISSUE TITLE
database.js: Page visibility indicator

### ISSUE THREAD

**Issue by joho1968** [WonderCMS/wondercms#301]
Looking at "menuItems" and "menuItems > subpages", I see there's "visibility". But looking at "pages" and "pages > subpages", there's no such setting. So I'm guessing this means I first have to traverse the "menuItems" and  "menuItems > subpages" to know if a given page is visible or not, if I need to figure that out for some reason. Is that correct? :thinking: 

It'd be nice if there was a "visibility" setting on each actual page node.

**Comment by robiso** [comment]
Yes, you are correct, this is possible by checking the menuItems themself.
We don't have visibility per page, as the visibility does not affect the visibility of the page. The visibility in menu items declares whether the page is visible in the menu or not (there are no "hidden" pages, and anyone can access those pages, even if their visibility is set to "invisible/hidden" :)

**Comment by joho1968** [comment]
OK. I think the possibility to hide a page may make sense, for example to "disable a page" temporarily instead of deleting it.

**Comment by robiso** [comment]
The solution looks something like:
- visibility is inherited from menuItem and a function that syncs the visibility between the pages and the menu items themselves.

More details in the PR: https://github.com/WonderCMS/wondercms/pull/327
